syntax = "proto3";

package fixbug;

// 一定要加下面这行，否则生成不了rpc方法
option cc_generic_services = true; 

message EchoRequest
{
    string message = 1;
}

message EchoResponse
{
    string message = 1;
}

message AddRequest
{
    int32 a = 1;
    int32 b = 2;
}

message AddResponse
{
    int32 result = 1;
}

service EchoServiceRpc
{
    rpc Echo(EchoRequest) returns(EchoResponse);
    rpc Add(AddRequest) returns(AddResponse);
}